src/3rd_party_repos.c

```diff
@@ -183,15 +183,15 @@ static int overwrite_repo_config_file(struct list *repos)
 	// open it in trunc mode as we re-write all the contents again
 	fp = fopen(repo_config_file_path, "w");
 	if (!fp) {
-		error("Impossible to write to file %s", repo_config_file_path);
+		error("Impossible to write to file %s\n", repo_config_file_path);
 		goto exit;
 	}
 
 	repo = repos;
 	while (repo) {
 		ret = write_repo(fp, repo->data);
 		if (ret < 0) {
-			error("Write error when trying to write to %s", repo_config_file_path);
+			error("Write error when trying to write to %s\n", repo_config_file_path);
```

Comment:
Error when trying to write to

Reply:
Same problem as before, starting the sentence with "error" will result in "Error": error when..."

Reply:
"Failed to write to %s" in that case